api/controllers.js

```diff
@@ -41,15 +41,23 @@ module.exports = {
   },
 
   deletePhoto: (req, res) => {
-    res.send(200);
+    if (!req.query.chatId || !req.query.messageId) {
```

Comment:
I appreciate the response letting you know what was missing instead of something generic